### ISSUE TITLE
Update Location Update even when the App is Active

### ISSUE THREAD

**Issue by siddharthcxor** [pmwisdom/cordova-background-geolocation-services#83]
This is a great plugin. But this would be a feature request. The plugin provide GPS data only when in background. However, it may be required to keep on sending data every 3 or 9 sec even when the app view is active.   

If there is something that can be done around it. Need access to GPS data constantly every 3 seconds whether the app is in foreground or background.

**Comment by veer0318** [comment]
I totally agree, that would be a great feature. Is something like this easy to implement?
